practice-exam/practice-exam-5.md

```diff
@@ -45,6 +45,9 @@ If this practice exam has been helpful to you please share it with others and re
 
     <details markdown=1><summary markdown='span'>Answer</summary>
       Correct answer: C
+      `This is because of the following`
+      `Pricing is per instance-hour consumed for each instance, from the time an instance is launched until it is terminated or stopped. Each partial instance-hour consumed will be billed per-second for Linux, Windows, Windows with SQL Enterprise, Windows with SQL Standard, and Windows with SQL Web Instances, and as a full hour for all other instance types.`
```

Comment:
Yes. 

Please don't create PR for confirmation. you can make an issue for this.